tests/unit/data/data_pipeline/test_round_robin.py

```diff
@@ -60,6 +60,38 @@ def test_op_works_when_infinite_pipeline_is_specified(self) -> None:
 
             pipeline.reset()
 
+    def test_op_works_when_infinite_pipeline_is_specified(self) -> None:
+        pipeline1 = read_sequence([1, 2, 3, 4]).and_return()
+        pipeline2 = read_sequence([0]).repeat().and_return()
+        pipeline3 = read_sequence([0, 2, 4, 6]).and_return()
+
+        pipeline = DataPipeline.round_robin(
+            [pipeline1, pipeline2, pipeline3]
+        ).and_return()
+
+        for _ in range(2):
+            it = iter(pipeline)
+
+            [next(it) for i in range(15)] == [
```

Comment:
Nit: Can you please have the entire list in a single line and use `#fmt: skip` to instruct black to ignore this line.